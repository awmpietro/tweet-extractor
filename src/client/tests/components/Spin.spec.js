import React from 'react';
import Spin from '../../components/Spin/Spin';
import renderer from 'react-test-renderer';

describe('Spin', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<Spin />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});