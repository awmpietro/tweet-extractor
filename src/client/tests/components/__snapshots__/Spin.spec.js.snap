// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spin snapshot renders 1`] = `
<div
  className="loading"
>
  Loadingâ€¦
</div>
`;
